tests/lib/nm/ipv4_test.py

```diff
@@ -29,15 +29,16 @@ def NM_mock():
 
 
 def test_create_setting_without_config(NM_mock):
-    ipv4_setting = nm.ipv4.create_setting(config=None)
+    ipv4_setting = nm.ipv4.create_setting(config=None, base_remote_conn=None)
```

Comment:
this also needs to be updated to base_con_profile and the other occurences (see the travis failure) or run tox locally.